test/runtests.jl

```diff
@@ -1,5 +1,5 @@
 using Ratios
-using Base.Test
+using Test
```

Comment:
If you use `Compat.Test` here and add a dependency to Compat 0.33 in the REQUIRE file this will keep working on julia 0.6 too. (I think this is the only change where Compat is needed.)